docs/objgraph.txt

```diff
@@ -19,6 +19,7 @@ Statistics
 
 .. autofunction:: show_growth([limit=10, peak_stats={}, shortnames=True, file=sys.stdout])
 
+.. autofunction:: get_ids([skip_update=False, limit=10, sortby='deltas'])
```

Comment:
Oh my, I don't remember how my own documentation works!  I totally forgot to include `growth()` and perhaps a few other new functions in here!